kivystudio/widgets/codeinput/__init__.py

```diff
@@ -47,10 +47,7 @@ class InnerCodeInput(HoverBehavior, CodeInput):
     def __init__(self, **kwargs):
         super(InnerCodeInput, self).__init__(**kwargs)
         self.rightclick_dropdown = CodeInputDropDown(self)
-        try:
-            self.style_name = 'native_tweak'
-        except ValueError:
-            self.style_name = 'colorful'
```

Comment:
sorry for this native tweak problems it was a tweak i was working on and didn't push so it caused problems

Reply:
no problem Oga

Reply:
lets make a todo on the important stuffs to implement so we dont work on the same stuffs at the same time